assets/anim.scss

```diff
@@ -416,7 +416,7 @@
 
 .chat-emote-Hhhehhehe {
     animation: hhhehhehe-anim 500ms 1;
-    transform-origin: 0 75%;
+    //transform-origin: 0 75%;
```

Comment:
Remove if no longer needed.

Reply:
I wasn't sure, it didnt break anything on my machine but SWEATSTINY